installing/installing_aws/upi/installing-aws-multiarch-support-upi.adoc

```diff
@@ -0,0 +1,78 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upi-aws-multiarch-support"]
+= Installing an AWS cluster with the support for configuring multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
+:context: upi-aws-multiarch-support
+
+toc::[]
+
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures.
+
+[NOTE]
+====
+When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning pods to nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
```

Comment:
If we are keeping this, should we use the ocp doc instead of the k8s one?

https://docs.openshift.com/container-platform/4.16/post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.html

Reply:
Absolutely right!

I am not sure why k8s doc was preferred over the ocp docs in this case.
I have replaced it with "xref:../../../post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc#scheduling-workloads-on-clusters-with-multi-architecture-compute-machines[Scheduling workloads on clusters with multi-architecture compute machines]."